### PR TITLE
Add info about restore

### PR DESCRIPTION
**Pull request by ArveVM**
## Proposed change

Adding restore as functionality - and providing info on limited functionality in Core-/Docker-versions

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by ArveVM** [comment]
adding information of limitation of restore in container/core versions.

File: source/installation/index.html [hunk 1]
```diff
@@ -231,12 +231,17 @@ <h2>Compare Installation Methods</h2>
...
+        <td>✅</td><td>❌<sup>2</sup></td><td>❌<sup>2</sup></td><td>✅</td>
+    </tr>
     <tr>
         <td>Managed OS</td>
         <td>✅</td><td>❌</td><td>❌</td><td>❌</td>
     </tr>
 </table>
 
 <sub>1: Backups for Home Assistant Core and Home Assistant Container is provided by the <a href="/integrations/backup" target="_blank"><code>backup</code> integration</a>.</sub>
+<sub>2: Backups for Home Assistant Core and Home Assistant Container is either a tool to migrate to HAOS, or completely manual restore of the backup</sub>
```
Comment (frenck) [inline source/installation/index.html]:
```suggestion
<sub>2: Backups for Home Assistant Core and Home Assistant Container are either a tool to migrate to HAOS or a completely manual restore of the backup.</sub>
```

**Review by frenck — APPROVED** [review]
Thanks, @ArveVM 👍 

../Frenck

File: source/installation/index.html (hunk 1 above)
Comment (ArveVM) [reply source/installation/index.html]: Thanks for the assistance on proper spelling :)     

I see now that the "footnotes" look funny,,, 
- the link to the backup-integration is not formated as a link anymore - so i guess i broke that one :( 
- and the 2'nd footnote is not starting on a new line (as it should?).
Can you "just fix" that,, or should I have another try with a new PR ?

(newbie on contributing so just figuring out how things work)

**Pull request merged**